src/components/layout/ChatArea.tsx

```diff
@@ -175,7 +173,9 @@ const MessageItem: React.FC<{
             <div className="w-10" />
```

Comment:
This needs to be changed to `w-8` too in order to fix an indentation workaround lol
![](https://i.ibb.co/Zp1WxYrz/Screenshot-from-2025-05-12-00-02-49.png)